### PR TITLE
Share counts

### PR DESCRIPTION
**Pull request by rich-nguyen**
Fetches the share counts for a given campaign using Reqwest, and updates the 'latest analytics' state object.

screen shot 2017-11-23 at 10 56 24

### CODE REVIEW

File: public/services/CampaignsApi.js [hunk 1]
```diff
@@ -38,11 +39,51 @@ export function fetchLatestAnalytics() {
...
+    return new Promise(function(resolve) {
+      Reqwest(req)
+        .then(res => {
+          values.linkedInShares = res.count;
+          resolve();
+        })
+        .fail(resolve);
+    });
+  });
+  return Promise.all(linkedInShares)
```
Comment (kelvin-chappell) [inline public/services/CampaignsApi.js:76]: Wouldn't it be better to fetch the share counts in a batch job and cache them in dynamo?

Comment (rich-nguyen) [reply public/services/CampaignsApi.js:76]: I'm not that worried about performance, and the information is coming from an API we have direct access to. It seems simple to just access it?

Comment (kelvin-chappell) [reply public/services/CampaignsApi.js:76]: I thought there was a requirement to report on shares by week etc but if not it's not a problem.  Could be done later anyway if needed.

**Review by kelvin-chappell — APPROVED** [review]
👍

File: public/services/CampaignsApi.js (hunk 1 above)
Comment (LATaylor-guardian) [reply public/services/CampaignsApi.js:76]: @rich-nguyen @kelvin-chappell Maybe worth checking with Chloe or Hannah.

**Pull request merged**